internal/useragent/useragent.go

```diff
@@ -3,11 +3,12 @@ package useragent
 import (
 	"fmt"
 	"strings"
+
+	"github.com/pinecone-io/go-pinecone/internal"
 )
 
 func getPackageVersion() string {
-	// update at release time
-	return "v0.5.0"
+	return internal.Version
```

Comment:
This is the only fix to the code itself, and we're expecting the `internal.Version` value to be somewhat managed through the CI process.

Reply:
nice!